tests/integration/workflows/test_arxiv_workflow.py

```diff
@@ -233,3 +302,253 @@ def test_harvesting_arxiv_workflow_manual_accepted(
         obj = eng.processed_objects[0]
         # It was accepted
         assert obj.status == ObjectStatus.COMPLETED
+
+
+def test_set_document_type_and_refereed_all_proceedings_none_refereed(workflow_app, insert_journals_in_db, workflow):
```

Comment:
Here is happening the same as commented in #2938. Maybe we can avoid running workflows, just instantiating a workflow object, since you are asserting that calling `set_document_type_and_refereed` on a workflow object does the right thing.